argon2.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: argon2
+  version: '20190702'
+  epoch: 0
+  description: The password hash Argon2, winner of PHC
+  copyright:
+    - paths:
+        - '*'
+      attestation: TODO
+      license: Apache-2.0
+  dependencies:
+    runtime:
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    environment:
+      PREFIX: ${{targets.destdir}}
+      DESTDIR: ${{targets.destdir}}
+    packages:
+      - wolfi-base
+      - build-base
+      - git
```

Comment:
This part isn’t needed, the git checkout action will indicate it needs git in the build environment.